src/init.cpp

```diff
@@ -1519,18 +1519,19 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
 
     // ********************************************************* Step 10: data directory maintenance
 
-    // if pruning, unset the service bit and perform the initial blockstore prune
+    // if pruning, perform the initial blockstore prune
     // after any wallet rescanning has taken place.
     if (fPruneMode) {
-        LogPrintf("Unsetting NODE_NETWORK on prune mode\n");
```

Comment:
Changing the logging isn't refactoring, so I think it should come with a rationale and without a "refactor" label

Reply:
Agree, removed the "refactor" tag from both the PR description and the commit. There is no deeper rationale than the one already given in the PR description (i.e. "it's the more obvious and less confusing way").